frontends/web/src/routes/account/info/buyReceiveCTA.module.css

```diff
@@ -0,0 +1,15 @@
+.container {
+    display: flex;
+    justify-content: center;
+}
+.container > button:first-child {
+    margin-right: var(--space-half);
+}
```

Comment:
Made a small change on the css  adding a `:first-child` selector to fix the following margin issue:

Before:
![Screenshot 2023-02-13 at 07 08 20](https://user-images.githubusercontent.com/28676406/218384879-76218816-fd52-420c-ab64-657cb995b87e.png)


After:

![Screenshot 2023-02-13 at 07 08 10](https://user-images.githubusercontent.com/28676406/218384894-b07a11e4-5c68-4b3e-b287-1d2a2b9953b2.png)

Reply:
tiny nit: the line-height of subTitle looks a bit too small, but this class is used in different places as well so lets keep as is and look at it in a different PR. I am not sure if other components depend on `line-height: 1;` or if that is ok to change globally.